testing/scripts/functional-test.sh

```diff
@@ -8,3 +8,11 @@ yarn run prebuild
 node content build
 
 yarn workspace testing run test $@
+
+# Pretend you wanna fix some fixable flaws.
+# First in dry-run mode.
+node content build --quiet -f web/fixable_flaws --fix-flaws --fix-flaws-dry-run --fix-flaws-verbose
+# Now actually, do it
+node content build --quiet -f web/fixable_flaws --fix-flaws
+git status | grep fixable_flaws/index.html # will fail if not in the output
+git diff testing/content/files/en-us/web/fixable_flaws/index.html
```

Comment:
Hm, I don't like this. I think it's too blunt of a test. It really only tests if the build commands run without error and if at least some change was made (even if it's the wrong change).

It also reminds me that I don't like that we have to either run `node content build` manually before we run `yarn workspace testing run test` or else run the `testing/scripts/functional-test.sh` script.

I think that when we run `yarn workspace testing run test` it should be one-stop shopping. We should take advantage of `jest`'s `beforeAll` and `beforeEach` as appropriate, to call the CLI programmatically via `cli.parse(...)`, and then run the individual tests that use `cheerio` or `jest-pupetteer` to check the details. We could get rid of this script entirely. However, that's for another PR, which I'll work on as an alternative approach. I don't think this needs to change for this PR.

Reply:
I'll put something up and see how it feels. Then I could use [`git-js`](https://github.com/steveukx/git-js) instead of UNIX to assert that the diff was what I expected.